optimizely/config_manager.py

```diff
@@ -308,4 +314,15 @@ def _run(self):
     def start(self):
         """ Start the config manager and the thread to periodically fetch datafile. """
         if not self.is_running:
```

Comment:
I  think add a condition here, if it is already disposed, then don't need to start.

Reply:
Need to discuss.